docs/guide/dev/code_documentation/README.md

```diff
@@ -229,24 +233,24 @@ Typical:
 		// ...
 	};
 
-#### Details ####
+#### Details
 
 * The `@method` tag has to be used when it is not clear that a piece of code is a method (e.g. a closure returning a function was used or a reference to a function defined elsewhere) or when the method's name is not obvious.
 * Callback arguments may be defined. Also, if a method returns an object, its properties may be defined too &mdash; [read more](https://github.com/senchalabs/jsduck/wiki/@return).
 * Both `@return` and `@returns` are accepted, but use the latter one.
 
-#### Events ####
+#### Events
 
 To define an event instead of a method:
 
 * Use `@event` instead of `@method` &mdash; usually you will have to provide a name.
 * `@returns` is not accepted.
 
-## Miscellaneous Rules ##
+## Miscellaneous Rules
 
 * Always leave one blank line between the textual comment and the first tag.
 * Separate all blocks (paragraphs, code samples, etc.) with one blank line.
-* Code samples are indented with at least two tabs after `*` and remember &mdash; **no spaces** are to be used.
+* Code samples are wrapped with triple back-ticks <code>\`\`\`</code>, optionally followed with a language identifier &mdash; **no spaces** are to be used.
```

Comment:
I'd suggest "backtick", "back tick" and backquote/back quote are also popular :)